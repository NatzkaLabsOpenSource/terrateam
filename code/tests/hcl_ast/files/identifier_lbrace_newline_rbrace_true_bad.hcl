foo {
} 1
---
foo {
} "foo"
---
foo {
} true

