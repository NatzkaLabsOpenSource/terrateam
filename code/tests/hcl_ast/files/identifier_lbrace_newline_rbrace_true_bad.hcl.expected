Newline expected after closing brace for block.
